src/index.mjs

```diff
@@ -76,9 +76,12 @@ app
   .use(flash())
   .use(compress(compressConfig()))
   .use(koaSwagger(swaggerConfig())) // swagger-ui at /docs
-  .use(koaCash(koaCashConfig()))
   .use(etag()) // required for koa-cash to propertly set 304
 
+if (process.env.NODE_ENV !== 'test') {
```

Comment:
The only way I could quickly find to disable caching in the test environment is to never enable it in the first place. It turns out that setting the MAX_AGE variable only sets it if it's not defined. So have to check whether the cash exists in every instance (which makes it look like there's a lot of changes).

Reply:
Unfortunate that they don't support disabling the cache in some other way 😕